internal/tasks/common.go

```diff
@@ -1,5 +1,26 @@
 package tasks
 
-const (
-	ItemsPerPage = 1000
+import (
+	api "github.com/flightctl/flightctl/api/v1alpha1"
+	"github.com/flightctl/flightctl/internal/store/model"
+	"github.com/flightctl/flightctl/internal/util"
 )
+
+var ItemsPerPage int
```

Comment:
We should only use globals when necessary is this the case here? Global scope can introduce strange bugs and introduce race conditions.